README.md

```diff
@@ -1862,6 +1866,14 @@ func main() {
 
 ## Caveats
 
+### Dots in Service Names
+
+Using dots `.` in service names will conflict with the use of dots for [TAG
+delineation](https://github.com/hashicorp/consul-template#service) in the
+template. Dots already [interfere with using
+DNS](https://www.consul.io/docs/agent/services.html#service-and-tag-names-with-dns)
+for service names, so it is really just best to not use dots if possible.
```

Comment:
```suggestion
for service names, so we recommend avoiding dots wherever possible.
```

Reply:
That sounds much better. Updated. Thanks.